docs/en/index.rst

```diff
@@ -111,6 +119,13 @@ Create a Job class::
          */
         public static $maxAttempts = 3;
 
+        /**
+         * Whether there should be only one instance of the job queued simultaneously.
+         * 
+         * @var bool
+         */
+        public static $shouldBeUnique = false;
```

Comment:
I'm not sure if this should be in the template or if users should add the property as needed. It doesn't have to be present, absence is assumed false. And really, this applies to `maxAttempts` as well. Let me know if you have thoughts on this.

Reply:
I don't think they need to be in the template. Having them only in the docs is fine. If you wanted to get really fancy, you could add CLI options to the bake task.

Reply:
I like the CLI option idea, I threw that in for this. I'll do that for the failed jobs one later and separately.